frontend/src/components/projects/ProjectsListContent.js

```diff
@@ -0,0 +1,167 @@
+import React, {useRef} from "react";
+import {connect} from "react-redux";
+import {Link} from "react-router-dom";
+import {Button, Tag} from "antd";
+
+import AppPageHeader from "../AppPageHeader";
+import PageContent from "../PageContent";
+import PaginatedTable from "../PaginatedTable";
+import AddButton from "../AddButton";
+import ExportButton from "../ExportButton";
+
+import api, {withToken}  from "../../utils/api"
+
+import {listTable, setFilter, setFilterOption, clearFilters, setSortBy} from "../../modules/projects/actions";
+import {actionsToButtonList} from "../../utils/templateActions";
+import {withContainer, withIndividual} from "../../utils/withItem";
+import {PROJECT_FILTERS} from "../filters/descriptions";
+import getFilterProps from "../filters/getFilterProps";
+import getNFilters from "../filters/getNFilters";
+import FiltersWarning from "../filters/FiltersWarning";
+import mergedListQueryParams from "../../utils/mergedListQueryParams";
+
+const getTableColumns = () => [
+    {
+      title: "ID",
+      dataIndex: "id",
+      sorter: true,
+      width: 80,
+      render: (id, project) =>
+        <Link to={`/projects/${project.id}`}>
+          <div>{id}</div>
+        </Link>,
+    },
+    {
+      title: "Name",
+      dataIndex: "name",
+      sorter: true,
+      width: 80,
+      render: (name, project) =>
+        <Link to={`/projects/${project.id}`}>
+          <div>{name}</div>
+        </Link>,
+    },
+    {
+      title: "Principal Investigator",
+      dataIndex: "principal_investigator",
+      sorter: true,
+      width: 70,
+    },
+    {
+      title: "Requestor Name",
+      dataIndex: "requestor_name",
+      sorter: true,
+      align: "right",
```

Comment:
Text is usually aligned left (default) and the className do not seem appropriate.